firmware/common/ui_widget.hpp

```diff
@@ -378,41 +378,21 @@ class Checkbox : public Widget {
 class Button : public Widget {
 public:
 	std::function<void(Button&)> on_select { };
+	std::function<void(Button&)> on_touch_release { }; // Executed when releasing touch, after on_select.
```

Comment:
Idea / suggestion: Wouldn't it be better to just draw a rectangle, place the "PTT BUTTON" text on it, and just monitor touch events on those coordinates (if possible) ... maybe by looking into the touch calibration app, it is easy to figure it out. Again, if possible, in this way you would not need to modify anything inside widgets.cpp

Reply:
is there is an easy access (callbacks) to touch events like those?